direct/core/src/test/resources/log4j.properties

```diff
@@ -1,6 +1,6 @@
 # Root logger option
-log4j.rootLogger=INFO, stdout
-#log4j.rootLogger=DEBUG, stdout
```

Comment:
these changes don't belong in PRs, this is changed in literally every 2nd PR ...